.github/workflows/pravega-build.yml

```diff
@@ -14,7 +14,7 @@
 
 name: Java CI
 
-on: [push]
```

Comment:
now that we are here.. pull request obviously makes sense.  does 'push' here mean all pushes, even to dev branches?  is that what we want?  or for push - we want only when there is push to master?
so, pull_request and push to master (?)